web/models.py

```
@@ -2921,3 +2932,44 @@ class Discount(models.Model):
+        ("delivered", "Delivered"),
+        ("open", "Opened"),
+        ("click", "Clicked"),
+        ("bounce", "Bounced"),
+        ("dropped", "Dropped"),
+        ("spamreport", "Spam Report"),
+        ("unsubscribe", "Unsubscribed"),
+        ("group_unsubscribe", "Group Unsubscribed"),
+        ("group_resubscribe", "Group Resubscribed"),
+        ("blocked", "Blocked"),
+        ("other", "Other"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)